pom.xml

```diff
@@ -98,6 +98,7 @@
         <fasterxml.jackson-core.version>2.13.4</fasterxml.jackson-core.version>
         <fasterxml.jackson-databind.version>2.13.4.2</fasterxml.jackson-databind.version>
         <toxiproxy.java.version>2.1.7</toxiproxy.java.version>
+        <commons-codec.version>1.16.1</commons-codec.version>
```

Comment:
Maybe you can add it next to the `commons-compress.version`?

Reply:
Yeah, good suggestion. Will do. Thanks